### ISSUE TITLE
Define density classes for AABW regions and fundamental simulation settings

### ISSUE THREAD

**Issue by Midway-X** [Midway-X/Backward_AABW#1]
**This issue is used to decide the AABW region to release the particles.**

**Comment by Midway-X** [comment]
following

**Comment by Midway-X** [comment]
**My attempt1:**
Following the results in Morrison et al., 2020, I set 3 major AABW formation regions: Prydz Bay, Weddell Sea and Ross Sea + Adelie Sea.
For each regions, I set a surface particle releasing region and a deeper water releasing regions. Basically the these regions are ruled by lat and lon border and 1000m depth contour. The surface particle releasing regions are basically the red parts in Fig 1 from Morrison et al., 2020 while the deeper water releasing regions is the blue parts in in Fig 1 from Morrison et al., 2020.

**Comment by PaulSpence** [comment]
@yinghuan, glad to see the progress here! Should we consider seeding the AABW parcels based on a density criterion? Perhaps look at the "age" tracer in the model (new AABW is young, also shown in Morrison et al.) and identify the density of young AABW off the shelf for the seeding? @vtamsitt is this a bad idea?

**Comment by Midway-X** [comment]
**Some pilot experiments:**
Here I plot the result of some short time backward running which range from 1year to 2 years and start from surface or 500m depth.
In the map the scatter with white edge is particles' start time in forward time and filled scatter is particles' end time in forward time (hence initial position in backward mode). In the depth profile, dark colour means end point in forward time (so they are at surface or 500m depth).

![Slide6](https://user-images.githubusercontent.com/74391012/162335705-b780ddc3-875f-48ca-83de-442514f4d425.PNG)
![Slide7](https://user-images.githubusercontent.com/74391012/162335716-f70eed44-3834-4621-9bae-f2f174b10e3f.PNG)
![Slide8](https://user-images.githubusercontent.com/74391012/162335718-f4568541-9bc5-4814-8f32-da828f9d8f63.PNG)
![Slide9](https://user-images.githubusercontent.com/74391012/162335722-fa6368df-8705-448c-ab63-0b742c9771c0.PNG)
![Slide10](https://user-images.githubusercontent.com/74391012/162335725-7d7b9165-1a73-4573-b884-2b80ff7b9524.PNG)

**Comment by Midway-X** [comment]
VT and I had a discussion with CS about her AABW definition standard. Here are key points:
CS used offshore flux across the 1000m isobath to define the AABW export in 4 main regions which is similar with regions in Morrison et al 2020. Specifically, this method considers all offshore flux across this isobath from bottom to surface as the AABW export. Obviously, this definition is sorely depended on the velocity field and changed with time.
Based on her introduction, this method well defined the AABW export because its timeseries fits the surface water mass transformation (SMWT) well in these regions.

**Comment by PaulSpence** [comment]
@Midway-X Thanks for the plots above https://github.com/Midway-X/Backward_AABW/issues/1#issuecomment-1092299857

In the next version, can you please zoom into the regions of interest, it's too hard to see the details of the trajectories.

**Comment by PaulSpence** [comment]
Re: "Specifically, this method considers all offshore flux across this isobath from bottom to surface as the AABW export."

That seems an odd choice to me. Does it assume that all northward surface flow across the 1000m isobath in DSW regions (e.g. melt water at the surface far from most coastal polynya's) flows down to the abyss?

Is the idea to seed parcels in a curtain (0-1000 m depth) along the 1000m isobath in the DSW formation regions and then identify the subset that eventually reaches the abyss?

**Comment by Midway-X** [comment]
Just a new idea: Maybe we could release particles still from the 1000m isobath but based on density threshold not the velocity threshold? This is not relevant with the velocity threshold plan but about the density threshold plan. @PaulSpence @vtamsitt

**Comment by PaulSpence** [comment]
Or seed the 1000m isobath curtain, track parcels forward in time and tag those that become AABW (via neutral density definition), then trace those tagged parcels them backwards in time?

**Comment by Midway-X** [comment]
Yes, I think this is a great idea. Meanwhile, I think we could also try to add the offshore criterion to see whether it could give extra confidence for us (i.e., 1000 m isobath curtain + AABW threshold in forward mode + offshore velocity at 1000m isobath)?
